flagsmith-engine/organisations/models.ts

```diff
@@ -0,0 +1,25 @@
+export class OrganisationModel {
+    id: number;
+    name: string;
+    featureAnalytics: boolean;
+    stopServingFlags: boolean;
+    persistTraitData: boolean;
+
+    constructor(
+        id: number,
+        name: string,
+        featureAnalytics: boolean,
+        stopServingFlags: boolean,
+        persistTraitData: boolean
+    ) {
+        this.id = id;
+        this.name = name;
+        this.featureAnalytics = featureAnalytics;
+        this.stopServingFlags = stopServingFlags;
+        this.persistTraitData = persistTraitData;
+    }
+
+    get unique_slug() {
```

Comment:
Can this be changed to `uniqueSlug` or will that cause some serialization issues further down the line?